docs/index.rst

```diff
@@ -0,0 +1,367 @@
+testimony
+=========
+An approach to document test cases in the python source code
+------------------------------------------------------------
+
+Are you tired of managing your test cases in a test case management tool and your test code in a python automation framework?  Testimony can help to use the python automation framework as a test case repository tool.
+
+The project testimony is written to inspect and report on the python test cases.  There are several reporting features in this program.
+
+The parameter options are:
+
+1. print - List all test cases
+2. summary - Summary of number of automated cases vs. manual cases
+3. validate_docstring - Validate docstrings. This does the following:s
+
+- Reports test cases with invalid docstrings
+- Reports test cases with missing docstrings
+- Reports test cases that does not have minimal required docstrings (This will enforce that all test cases have a minimal set of docstrings). Currently testimony enforces Feature, Test, Assert as mandatory docstrings for each test case"
+
+4. bugs - Test cases affected by Bugs and the corresponding Bug list
+5. manual - List all manual test cases
+6. auto - List all auto test cases
+7. tags - Prints all test cases with the specified tags
+
+Note:
+
+1. testimony returns a non-zero error code when the test case docstrings does not follow the intended rules, returns zero otherwise.  This will help
+   testimony to be integrated easily with tools like travis.
+2. testimony also parses different folders under the given folder to verify the test files
+3. testimony also displays optional colored outputs when termcolor is installed
+
+Advantages
+----------
+1. Avoids a separate test case management tool to document test cases by using the python automation framework for the same
+2. Enforces test cases to follow a defined standard
+3. Runs in integration with tools like Travis to get a report as and when you check in code
+4. Saves a lot of time from the conventional way of writing test cases using a test management tool
+5. The test case information can be easily extracted from testimony and can be ported to any test management tool 
+
+Installation
+------------
+
+You could install testimony from PyPI using pip:
+
+::
```

Comment:
FYI, this can also be written as:

```rest
You could install testimony from PyPI using pip::

    pip install testimony
```

But there's absolutely no different at all in semantics. I'm only pointing out some syntactic sugar.